client/assets/components/navbar.scss

```diff
@@ -0,0 +1,171 @@
+.top-navbar {
+  height: 70px;
+  border-radius: 0;
+  background-color: #fff;
+  margin: 0;
+  border: 0;
+  .navbar-brand {
+    height: 70px;
+    width: 70px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+}
+.navbar-collapse.collapse {
+  display: flex !important;
+  height: 70px !important;
+}
+.top-navbar__right-container {
+  align-items: center;
+  justify-content: right;
+  padding: 0;
+  .nav-profile {
+    width: 183.5px;
+    &__list-menus {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      box-shadow: -1px 6px 6px #dedede;
+      border: 0;
+      border-radius: 0;
+      height: 9.125rem;
+      width: 10rem;
+      li {
+        height: 2.5rem;
+        width: 70%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin: 1em 0 1em 0;
+        &:first-child:after {
+          content: '';
+          position: absolute;
+          bottom: 50%;
+          width: 80%;
+          border-bottom: 1px solid #dedede;
+        }
+        a {
+          color: #ccc;
+          font-family: Abel;
```

Comment:
### Critical

We are making use of `Abel` throughout the project and I believe it has been defined at the top level style, there's no need referencing it here.

Reply:
@BolajiOlajide we aren't using `Abel`ide since you said we should aim to follow andela's style gu, `DINPro` is andela's sans serif font

Reply:
True. Though the font used for the mock is `Abel` due to some font-style ish with DINPro, but if we have one central point of font-definition then changing it won't be hard.

Right now I've seen about 4 font definitions in this project